utils/appliance.py

```diff
@@ -38,7 +36,9 @@
 RUNNING_UNDER_SPROUT = os.environ.get("RUNNING_UNDER_SPROUT", "false") != "false"
 # Do not import the whole stuff around
 if not RUNNING_UNDER_SPROUT:
-    from cfme.configure.configuration import set_server_roles, get_server_roles
```

Comment:
This is to streamline it when we are not running under sprout, but is there protection on the methods that call using those imports?